app/src/main/res/values-zh/strings.xml

```diff
@@ -10,91 +10,116 @@
 	<string name="action_credits">关于</string>
 
 	<!-- List -->
-	<string name="inaccessible">(不可达)</string>
+	<string name="inaccessible">(无法通过)</string>
 	<string name="go_home_folder">有问题吗？点击返回首页</string>
-	<string name="play_folder">在这儿看幻灯片</string>
+	<string name="play_folder">在此运行幻灯秀</string>
 
 	<!-- Slideshow -->
 	<string name="title_activity_image">&appname;</string>
 
 	<string name="delete_button">删除</string>
 	<string name="delete_dialog_title">删除照片</string>
-	<string name="delete_dialog_message">有把握吗？</string>
-	<string name="image_deleted">照片被删除了</string>
-	<string name="image_not_deleted">不能删除照片</string>
+	<string name="delete_dialog_message">确定删除？</string>
+	<string name="image_deleted">照片已被删除</string>
+	<string name="image_not_deleted">照片未能删除</string>
 
 	<string name="share_button">分享</string>
 	<string name="share_via">通过&#8230;分享</string>
 
 	<string name="image_detail_dimensions">尺寸: %1$dx%2$d</string>
 	<string name="image_detail_size">文件大小: %1$s</string>
-	<string name="image_detail_modified">修改了: %1$s</string>
+	<string name="image_detail_modified">文件修改: %1$s</string>
+        <string name="toast_no_files">无文件显示</string>
+
+	<string name="long_loading_warning">%1$s 加载时间过长.</string>
+	<string name="long_loading_skipping">%1$s 加载时间过长. 跳过.</string>
+	<string name="long_loading_skip_action">跳过</string>
 
 	<!-- Image descriptions -->
 	<string name="image_description">照片</string>
-	<string name="logo_description">商标</string>
+	<string name="logo_description">Logo</string>
 
 	<!-- Credits -->
-	<string name="title_activity_credits">关于 &appname;</string>
-	<string name="credits_creator">被 <a href="https://michael.standen.link">&author;</a></string>
-	<string name="credits_content1">幻灯片是一个简单的应用软件为了在手机上看照片。</string>
-	<string name="credits_content2">因为没有编钟和口哨，这个应用软件是市场最简单的幻灯片应用软件。</string>
+	<string name="title_activity_credits">关于 &软件名;</string>
+	<string name="credits_creator">由 <a href="https://michael.standen.link">&author;</a></string>
+	<string name="credits_content1">幻灯秀是一个在手机上看照片的简洁的应用。</string>
+	<string name="credits_content2">没有浮夸的装饰，本应用是市面上最简洁的幻灯片应用。</string>
 	<string name="credits_content3">在<a href="https://github.com/Screaminghawk/android-slideshow">Github</a>
-		你可以看源代码，扽入故障，和请求特点。
+		上你可以查看源代码，报告故障，和提出建议。
 	</string>
-	<string name="credits_content4">这个应用软件全又免费又开源；有
+	<string name="credits_content4">这个应用软件完全免费且开源；遵循
 		<a href="https://tldrlegal.com/license/mit-license">MIT 许可证</a>。
 	</string>
-	<string name="credits_content5">谢谢<a href="https://github.com/psyanite">Yan Tsui</a>
-		为了测试和设计&appname;图标.
+	<string name="credits_content5">感谢<a href="https://github.com/psyanite">Yan Tsui</a>
+		对测试和设计的贡献&appname;icon.
 	</string>
-	<string name="credits_content6"> <a href="https://github.com/wardbradt">Ward Bradt</a>
-		把&appname;从英文翻译成中文。
+	<string name="credits_content6"> 此版本翻译&appname;由<a href="https://github.com/wardbradt">Liuxi-G</a>
```

Comment:
Could you please keep the original translator here and add your own `<a href="your_website_link">Luixi-G</a>` as well. I'd like to keep both translators, even if you are changing most of it :)

Reply:
Hate to say this but as a native chinese speaker the original translation seems to be google translated,sure if you want to keep the original one,I can send a new pull request.